react-exercise/src/types/filter.ts

```diff
@@ -0,0 +1 @@
+export type FilterType = "Cities" | "Age" | null;
```

Comment:
I wan't sure if it's appropriate to add null like this so I can initialize the state with null and still declare the filter state value as FilterType. If it is not the way to handle it- please let me know @danafl